test/e2e/csv_e2e_test.go

```diff
@@ -312,6 +313,85 @@ func TestCreateCSVWithUnmetPermissionsCRD(t *testing.T) {
 
 }
 
+func TestCreateCSVWithNotReadyCRD(t *testing.T) {
+	defer cleaner.NotifyTestComplete(t, true)
+
+	c := newKubeClient(t)
+	crc := newCRClient(t)
+
+	depName := genName("dep-")
+
+	// Create dependency first (CRD)
+	crdPlural := genName("ins2")
+	crdName := crdPlural + ".cluster.com"
+	crd := extv1beta1.CustomResourceDefinition{
```

Comment:
I suggest adding an ampersand so you're passing a pointer to `UpdateStatus` below: `&extv1beta1.CustomResourceDefinition{`.  That should resolve the compiler error.